core/src/main/java/io/confluent/rest/Application.java

```diff
@@ -191,7 +191,16 @@ public Server createServer() throws ServletException {
     // The configuration for the JAX-RS REST service
     ResourceConfig resourceConfig = new ResourceConfig();
 
-    Map<String, String> configuredTags = getConfiguration().getMap(RestConfig.METRICS_TAGS_CONFIG);
+    Map<String, String> configuredTags = new HashMap<>();
+    // This is copied from the old MAP implementation from cp ConfigDef.Type.MAP
+    List<String> metricsTags = getConfiguration().getList(RestConfig.METRICS_TAGS_CONFIG);
+    for (String entry : getConfiguration().getList(RestConfig.METRICS_TAGS_CONFIG)) {
```

Comment:
Probably worth putting this into a utility method.

Reply:
also added a test